libs/concepts/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2019 The STE||AR-Group
+#
+# Distributed under the Boost Software License, Version 1.0. (See accompanying
+# file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+cmake_minimum_required(VERSION 3.3.2 FATAL_ERROR)
+
+set(concepts_headers
+  hpx/concepts/concepts.hpp
+  hpx/concepts/has_member_xxx.hpp
+  hpx/concepts/has_xxx.hpp
+)
+
+set(concepts_compat_headers
+  hpx/traits/concepts.hpp
+  hpx/traits/has_member_xxx.hpp
+  hpx/traits/has_xxx.hpp
+)
+
+set(concepts_sources)
+
+include(HPX_AddModule)
+add_hpx_module(concepts
+  COMPATIBILITY_HEADERS ON
+  DEPRECATION_WARNINGS
+  FORCE_LINKING_GEN
+  GLOBAL_HEADER_GEN OFF
```

Comment:
Any particular reason why we don't want the global header? It shouldn't clash as far as I can tell.

Reply:
nope ^^